wiki/osu!api/de.md

```diff
@@ -0,0 +1,11 @@
+# osu!api
+
+osu! hat eine eigene und öffentliche API, die Entwickler nutzen können, um die osu! Plattform unterstützen zu können und zu erweitern. Es kann nur privat genutzt werden, die auf Anfrage beantragt werden kann. Aber nun kann dies jeder nutzen und es bleibt, solange ihr Account existiert, bestehen.
+
+Du kannst eine Dokumentation von der API auf der [osu-api Wiki](https://github.com/ppy/osu-api/wiki) finden. Die aktuelle Version der API nutzt private Keys für die Authentifizierung der Anfragen. Du kannst dir einen API Key auf der [Beantragungs-Seite](https://osu.ppy.sh/p/api) von osu! beantragen.
+
+Denke aber daran, dass die Dokumentation der osu!api nicht vollständig oder umfangreich ist, und die API nicht aktiv entwickelt wird. Eine neue und bessere API ist gerade in Arbeit und zu finden auf [osu-web](https://github.com/ppy/osu-web).
```

Comment:
Das "may" als Relativierung wurde in der Übersetzung ausgelassen, welches mMn sehr wichtig ist. Würde den zweiten Satz kürzen inden du aus "zu finden auf [osu-web]" einfach "im [osu-web] machst.